types/firestore.d.ts

```diff
@@ -613,12 +616,24 @@ declare namespace FirebaseFirestore {
   }
 
   /**
-   * An options object that can be used to disable request throttling in
-   * BulkWriter.
+   * An options object to configure throttling on BulkWriter.
    */
   export interface BulkWriterOptions {
-    /** Whether to disable throttling. */
-    readonly disableThrottling?: boolean;
+    /**
+     * Whether to disable or configure throttling. By default, throttling is
+     * enabled.
+     *
+     * @param initialOpsPerSecond The initial maximum number of operations per
+     * second allowed by the throttler. If this field is not set, the default
+     * is 500 operations per second.
+     * @param maxOpsPerSecond The maximum number of operations per second
+     * allowed by the throttler. If this field is set, the throttler's allowed
+     * operations per second does not ramp up past the specified operations per
+     * second.
+     */
+    readonly throttling?:
+      | boolean
+      | {initialOpsPerSecond?: number; maxOpsPerSecond?: number};
```

Comment:
Thanks for the update. Did you check with @Feiyang1 if he thinks this is a useable API?

Reply:
This looks fine to me. Autocomplete will help people figure it out. Currently we are only describing the config params in the comment. I would make it clear in the comment that it can be either a boolean or a config object (boolean to enable/disable, config to enable and set config values).

Reply:
Was going to ask him after, but he jumped in :)

Reply:
Do you also want to update the comment per Fei's suggestion?

Reply:
I thought I had already added some more to the comments, but this time around I also explicitly stated that setting it to false disables throttling.